server/app.py

```diff
@@ -118,12 +162,54 @@ def create_club():
 
 @app.route('/club', methods=['PUT'])
 def modify_club():
-    return 'UNIMPLEMENTED'
+    if request.mimetype != 'application/json':
+        raise Exception('Content-Type is not "application/json".')
+    j = request.get_json()
+    Session, engine = dbconnect(db_options)
+    session = Session()
+    clubID = j.get('id')
+    matchingClubs = session.query(Club).filter(Club.id == clubID).one_or_none()
+    if matchingClub:
+        matchingClub.name=j.get('name')
```

Comment:
still the same feedback about being able to pick and choose which fields you're modifying